.goreleaser.yml

```diff
@@ -31,46 +33,31 @@ brews:
   github:
     owner: mongodb-labs
     name: homebrew-mongocli
-  url_template: "https://github.com/mongodb/mongocli/releases/download/{{ .Tag }}/{{ .ArtifactName }}"
-  commit_author:
-    name: gustavo.bazan
-    email: gustavo.bazan@mongodb.com
-  homepage: "https://github.com/mongodb/mongocli"
+  skip_upload: auto
   description: The MongoDB Command Line Interface (mongocli) is a tool for managing your MongoDB cloud services, like MongoDB Atlas, MongoDB Cloud Manager, and MongoDB Ops Manager.
```

Comment:
[q] Are these descriptions still needed as you've moved them into `.goreleaseer_evg.yml`? Just wondering can we reduce the reuse of magic string and handle it all in the linked script.

Reply:
yes but I decided against this, the whole point was about naming and I think I going to do this in evg